qdrant_client/conversions/conversion.py

```diff
@@ -1955,23 +1955,26 @@ def convert_batch_vector_struct(
     @classmethod
     def convert_named_vector_struct(
         cls, model: rest.NamedVectorStruct
-    ) -> Tuple[List[float], Optional[str]]:
+    ) -> Tuple[List[float], Optional[grpc.SparseIndices], Optional[str]]:
         if isinstance(model, list):
```

Comment:
I am a little bit confused by our naming...

Why do we have `NamedVectorStruct` which can be a `List[StrictFloat]` :(

Code is correct though 👍

Reply:
Anonymous named vector #legacy